simulator/src/canvasApi.js

```
@@ -110,7 +110,7 @@ export function dots(dots = true, transparentBackground = false, force = false)
     var ctx = backgroundArea.context;
     ctx.beginPath();
     backgroundArea.clear();
-    ctx.strokeStyle = colors["canvas_stroke"];
+   ctx.strokeStyle = colors["canvas_stroke"];
```

ESLint

[error] Expected indentation of 4 spaces but found 3.

(indent)